low-level/sensors/impl/generic/node_hw_alerts_info.py

```diff
@@ -0,0 +1,116 @@
+# Copyright (c) 2001-2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+from collections import namedtuple
+from framework.base.sspl_constants import DEFAULT_RECOMMENDATION
+
+Alert = namedtuple('AlertInfo',
+                   ['type', 'severity', 'description',
+                    'impact', 'recommendation'])
+
+alert_for_event = {
+    "Fan": {
+        **dict.fromkeys(["Fully Redundant", "Fully Redundant ()"], {
+            "Asserted": Alert(
+                "fault_resolved", "informational",
+                "'{} Sensor' has recovered from a failure. [{}]",
+                "None", "None"),
+            "Deasserted": Alert(
+                "fault", "critical",
+                "'{} Sensor', is Fully redundant . [{}]",
+                "Fan efficiency will be impacted",
+                DEFAULT_RECOMMENDATION)
+        }),
+        **dict.fromkeys(["Lower Non-Recoverable", "Lower Non-Recoverable ()"], {
+            "Asserted": Alert(
+                "fault", "critical",
+                "Lower Non-Recoverable threshold breached for '{}'. [{}]",
```

Comment:
How this description , impact, recommendation helping here exactly ?

Need something like below for every case
description -> Fan {} RPM Lower Non-Recoverable threshold breached, its below acceptable limits.
impact -> Other System Fans may increase speed to maintain optimal server temperature.
recommendation -> System should recover by itself overtime. However, If issue persists for long, contact Seagate support

Reply:
Ack.
Improvised the field description